plugin.py

```diff
@@ -208,6 +208,15 @@ def on_pre_server_command(
             return True
         return False
 
+    def on_workspace_configuration(self, params: Dict, configuration: Any) -> Any:
+        if "section" in params and "scopeUri" in params and params["section"] in JDTLS_CONFIG_TO_SUBLIME_SETTING:
```

Comment:
I'm assuming that you've checked it and the server explicitly requests sections `java.format.tabSize` and `java.format.insertSpaces` rather than the parent `java.format`, for example? I would think that the latter would be more reasonable and efficient (also a bit more tricky handle on our side).

Reply:
Yes tested it and looked into the vscode extension code.